appserver/tests/payara-samples/samples/pom.xml

```diff
@@ -41,6 +41,7 @@
         <module>rolesallowed-unprotected-methods</module>
         <module>microprofile-rc-ft</module>
         <module>multiple-keystores</module>
+        <module>security</module>
```

Comment:
Minor point - this is an exceedingly generic name for this test which doesn't actually describe what it's doing - it looks more like an aggregator module name.

I'd rename it to client-cert-validator